### DISCUSSION TITLE
How to change Button font-size in the theme?

### DISCUSSION THREAD

**Discussion by hornetDC** [microsoft/fluentui#27233] [Q&A] [answered]
Hi. I can't figure out how to change the default (14px) button size in the theme.

```
const buttonStyles: IButtonStyles = {
  root: {
    borderRadius: 4,
    fontSize: "16px", // doesn't work
  },
  label: {
    fontWeight: "16px", // doesn't work
  },
};

const fluent: PartialTheme = {
  defaultFontStyle: { fontFamily: "Source Sans Pro" },
  components: {
    DefaultButton: {
      styles: buttonStyles,
    },
  },
};
```

**Comment by khmakoto** [comment, marked as answer]
Hey @hornetDC, something like what you are doing should work. You can see it working [in this codesandbox](https://codesandbox.io/s/theme-that-changes-button-fontsize-4se5yo?file=/src/ThemeProviderExample.tsx).

Let us know if there is anything else we can do to help.

**Reply by hornetDC** [reply]
You are right, seems something else is overriding my button. We use our own ui library built on top of fluent, button from our lib has a different root-xxx class.
![image](https://user-images.githubusercontent.com/31066977/225711100-e8ebf119-2cd7-42c5-bbd6-ce6f517de36c.png)

**Reply by khmakoto** [reply]
That last number is auto-generated by our styling+theming library, so I wouldn't say that it necessarily means that you don't have it. However, if you were expecting the styles for those 2 buttons to be the same, then there is something definitely overriding at least one of those.

**Reply by hornetDC** [reply]
Button from our library has a font override of it's own, mystery solved. Thanks!
![image](https://user-images.githubusercontent.com/31066977/225717695-92a8f47e-60be-4b2b-964b-172a4fe27026.png)

**Reply by khmakoto** [reply]
No prob! Glad you could track that down!
